src/components/Popover/index.tsx

```diff
@@ -18,6 +18,33 @@ type PropsType = {
     renderContent(): JSX.Element | string;
 };
 
+/**
+ * Tooltip component that is positioned around the component it's wrapped around.
```

Comment:
There's a lot of around in this sentence 😄. To avoid confusion maybe this could be:
```suggestion
 * Tooltip component that is positioned around the component it wraps.
```